pkg/utils/webhook/webhook.go

```diff
@@ -104,9 +104,12 @@ func (iv *InstallationValidator) handlePrivate(ctx context.Context, req admissio
 	}
 
 	if errs := validation.ValidateInstallation(inst); len(errs) > 0 {
-		return admission.Denied(errs.ToAggregate().Error())
+		errMsg := errs.ToAggregate().Error()
+		logger.Info("Denied request", lc.KeyError, errMsg)
+		return admission.Denied(errMsg)
 	}
 
+	logger.Info("Allowed request")
```

Comment:
Might this log to info flood the logs? Perhaps in the good case debug is sufficient.